pkgs/development/python-modules/nbqa/default.nix

```diff
@@ -1,18 +1,33 @@
 {
   lib,
-  python3,
+  buildPythonPackage,
```

Comment:
Could you split this into three commits?

1. `nixfmt`
2. Manual refactoring
3. Moving the package

Reply:
I can do an initial `nixfmt`, but most of the manual refactoring was done because of the switch to the way that `python-modules` works.